src/Controller/BackendController.php

```diff
@@ -116,32 +103,33 @@ public function requestPasswordAction()
         $template->language = $GLOBALS['TL_LANGUAGE'];
```

Comment:
Geht nicht in Contao 5 (https://github.com/contao/contao/blob/4.13/DEPRECATED.md)